### ISSUE TITLE
Getting blank output, not quite sure why?

### ISSUE THREAD

**Issue by lloydmeta** [BurntSushi/cargo-benchcmp#26]
Hi there,

Thanks for writing this cool tool :)  I'm new to Rust, so please excuse any stupid questions.

I'm trying to use benchcmp with my lib, [Frunk](https://github.com/lloydmeta/frunk), using the following:

``` shell
$ rustup run nightly cargo bench > benches-control && rustup run nightly cargo bench > benches-variable
#.. then 
$ cargo benchcmp benches-control benches-variable
```

I understand that benchmarking on the same branch will yield zero (or close to) differences, and my intention is to use benchcmp to compare performance before and after applying a PR, so I'll switch branches in between during actual usage.

Unfortunately the second command yields no output and I'm not sure why ? My benches are [here](https://github.com/lloydmeta/frunk/tree/master/benches), located in a separate directory.

Thanks in advance for your help,

Lloyd

**Comment by BurntSushi** [comment]
Can you please include the benchmark outputs so that we can more easily
reproduce your problem?

**Comment by lloydmeta** [comment]
Sure thing:
- [Control results](https://gist.github.com/lloydmeta/2d83e2ec2566cad752e2156d3e37bd1f)
- [Variable results](https://gist.github.com/lloydmeta/a8cad440458860fc8ef4574d0c30b9f5)

**Comment by BurntSushi** [comment]
I think this should be fixed in `0.1.5`. It looks like this was a regression introduced by #21. I've backed that change out.

**Comment by lloydmeta** [comment]
@BurntSushi thanks for that. Just gave 0.1.5 a try and can confirm it works fine.
